src/EA.Weee.Requests/AatfEvidence/GetAatfNotesRequest.cs

```diff
@@ -21,11 +21,15 @@ public class GetAatfNotesRequest : EvidenceNoteFilterBaseRequest
 
         public DateTime? EndDateSubmitted { get; set; }
 
+        public int? SelectedComplianceYear { get; set; }
+
+        public DateTime? CurrentDate { get; set; }
+
         public GetAatfNotesRequest(Guid organisationId, 
             Guid aatfId, 
             List<NoteStatus> allowedStatuses,
             string searchRef, Guid? recipient, WasteType? wasteType, NoteStatus? noteStatusFilter,
-            DateTime? startDateSubmitted, DateTime? endDateSubmitted)
+            DateTime? startDateSubmitted, DateTime? endDateSubmitted, int? selectedComplianceYear, DateTime? currentDate)
```

Comment:
Not sure we need to pass the current date to the request?

Reply:
Previous experiences says that int? selectedComplianceYear should be named complianceYear otherwise the the json serializer wont be able to set the properties